tests/unit/module_utils/test_files/networking-definition-valid-all-tools-full-map-out.json

```diff
@@ -365,4 +373,4 @@
     }
   },
   "routers": {}
-}
+}
```

Comment:
I will fix the absent newline at the end of these files before opening a real PR